pkg/nuctl/command/renderers.go

```diff
@@ -0,0 +1,153 @@
+package command
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/nuclio/nuclio/pkg/common"
+	"github.com/nuclio/nuclio/pkg/platform"
+	"github.com/nuclio/nuclio/pkg/renderer"
+
+	"golang.org/x/sync/errgroup"
+)
+
+func formatFunctionIngresses(function platform.Function) string {
+	var formattedIngresses string
+
+	ingresses := function.GetIngresses()
+
+	for _, ingress := range ingresses {
+		host := ingress.Host
+		if host != "" {
+			host += ":<port>"
+		}
+
+		for _, path := range ingress.Paths {
+			formattedIngresses += fmt.Sprintf("%s%s, ", host, path)
+		}
+	}
+
+	// add default ingress
+	formattedIngresses += fmt.Sprintf("/%s/%s",
+		function.GetConfig().Meta.Name,
+		function.GetVersion())
+
+	return formattedIngresses
```

Comment:
`formatFunctionIngresses` doesn't look like it belongs here
Maybe this should be some common helpers under nuctl/command ?
@liranbg  ?

Reply:
not opposed to changing to common, just like using more explicit names, usually

Reply:
as long as it only renders, fine, if contain more -> common/renderers & common/helpers